client/ayon_core/modules/deadline/plugins/publish/submit_houdini_render_deadline.py

```diff
@@ -96,7 +97,9 @@ class HoudiniSubmitDeadline(
                 "arnold_rop",
                 "mantra_rop",
                 "karma_rop",
-                "vray_rop"]
+                "vray_rop",
+                "usdrender"]
```

Comment:
How come this is new in this PR by the way? Wasn't submission to deadline already working? :/

Reply:
I've added it upstream in my branch as well - seemed like a refactor issue that it got lost?

Reply:
Ah, nevermind. Found it - it's because it has its own class at the bottom of this file.
This family added now is also why you're getting double submissions @MustafaJafar